src/posix_storage.cpp

```diff
@@ -512,10 +515,18 @@ namespace aux {
 	{
 		std::string const fn = files().file_path(idx, m_save_path);
 
-		char const* mode_str = (mode & open_mode::write)
+		const auto mode_str = (mode & open_mode::write)
```

Comment:
```suggestion
		auto const* mode_str = (mode & open_mode::write)
```

right now the arrays decay to a pointer via the ternary operator. It should be explicit.